ThunderCloud/QuizAreaSelectionViewController.swift

```diff
@@ -68,42 +66,55 @@ open class QuizAreaSelectionViewController: UIViewController, QuizQuestionViewCo
         let relativeLocation = CGPoint(x: location.x / imageView.bounds.width, y: location.y / imageView.bounds.height)
         question?.answer = relativeLocation
         
-        if let circle = circleLayer {
-            circle.removeFromSuperlayer()
-        }
+        [circleLayer, circleInnerLayer, circleShadowLayer].forEach({ $0?.removeFromSuperlayer() })
```

Comment:
Do we need to be removing and adding again?

Reply:
Not 100% sure, just went with what was there previously... probably no need to, but also no need to *not* if you get what I mean? There's also a possibility this was added in previously because there was an issue if you didn't remove it... We'll never know (As there wasn't a comment there) :P